azkaban-common/src/test/java/azkaban/project/ProjectManagerTest.java

```diff
@@ -0,0 +1,162 @@
+package azkaban.project;
+
+import azkaban.user.User;
+import azkaban.utils.Props;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class ProjectManagerTest {
+
+  private File workingDir;
+
+  @Before
+  public void setUp() throws Exception {
+    System.out.println("projectManagerTest: create temp dir");
+    synchronized (this) {
+      workingDir = new File("_AzkabanTestDir_" + System.currentTimeMillis());
+      if (workingDir.exists()) {
+        FileUtils.deleteDirectory(workingDir);
+      }
+      workingDir.mkdirs();
+    }
+  }
+
+  @After
+  public void tearDown() throws IOException {
+    System.out.println("Teardown temp dir");
+    synchronized (this) {
+      if (workingDir != null) {
+        FileUtils.deleteDirectory(workingDir);
+        workingDir = null;
+      }
+    }
+  }
+
+  @Test
+  public void testCreateProject() throws ProjectManagerException {
+    System.out.println("projectManagerTest: testCreateProject");
+    ProjectManager manager = createProjectManager();
+    String projectName = "myTestCreateProject";
+    String projectDescription = "This is to test creating a project";
```

Comment:
For different tests, the project names and descriptions are different. Might not be easy to combine those together?